docs/src/main/paradox/bootstrap/kubernetes.md

```diff
@@ -0,0 +1,149 @@
+# Kubernetes
+
+As of Akka 2.5.15 and Akka-Management 0.18 the recommended way to run Akka Cluster in Kubernetes is to:
+
+* Use Akka Bootstrap with `akka-dns` with cluster formation via DNS SRV records
+* Use a headless service for internal for Akka management/bootstrap so that readiness probes for prod traffic don't interfere with bootstrap 
+* If required use a separate service and/or ingress for user facing endpoints e.g. HTTP gRPC
+
+# Kubernetes Services
+
+For Akka Cluster / Management use a headless service. This allows the solution to not be coupled to k8s as well
+as there is no use case for load balancing across management/remoting ports.
+Set endpoints to be published before readiness checks pass as these endpoints are required to bootstrap the Cluster
+and make the application ready. 
+
+
+```
+apiVersion: v1
+kind: Service
+metadata:
+  labels:
+    appName: "akka-cluster-kubernetes"
+  annotations:
+    service.alpha.kubernetes.io/tolerate-unready-endpoints: "true"
+  name: "akka-cluster-kubernetes"
+spec:
+  ports:
+    - name: management
+      port: 8558
+      protocol: TCP
+      targetPort: 8558
+    - name: remoting
+      port: 2552
+      protocol: TCP
+      targetPort: 2552
+  selector:
+    appName: "akka-cluster-kubernetes"
+  clusterIP: None
+  publishNotReadyAddresses: true
+```
+
+Note there are currently two ways to set that addressed should be published if not ready, the initial way via an annotation 
+`service.alpha.kubernetes.io/tolerate-unready-endpoints` and via the new officially supported way as the property `publishNotReadyAddresses`.
+Set both as depending on your DNS solution it may have not migrated from the annotation to the property.
+
+This will result in SRV records being published for the service that contain the nodes that are not ready. This allows
+bootstrap to find them and form the cluster thus making them ready.
+
+Then to configure your application:
+
+```
+akka {
+  loglevel = DEBUG
+
+  io.dns.resolver = async-dns
+
+  actor {
+    provider = "cluster"
+  }
+
+  management {
+    cluster.bootstrap {
+      contact-point-discovery {
+        port-name = "management" # name of the port in the headless service
+        protocol = "tcp" # protocol in the headless service
+        service-name = "akka-cluster-kubernetes" # headless service name
+        service-namespace = "default.svc.cluster.local" # your namespace/cluster name
+      }
+    }
+
+    http {
+      port = 8558
+      bind-hostname = "0.0.0.0"
+    }
+  }
+
+  remote {
+    netty.tcp {
+      port = 2552
+    }
+  }
+
+  discovery {
+    method = akka-dns
+  }
+}
+```
+
+The same configuration will work for any environment that has an SRV record for your Akka Clustered application. 
+
+For prod traffic e.g. HTTP use a regular service or an alternative ingress mechanism. 
```

Comment:
Maybe add 2 subsection headers in this section ('Headless service' and 'Exposed service'?)

Reply:
:+1: